pyodide_build/xbuildenv.py

```diff
@@ -265,15 +265,28 @@ def _install_cross_build_packages(
         """
         host_site_packages = self._host_site_packages_dir(xbuildenv_pyodide_root)
         host_site_packages.mkdir(exist_ok=True, parents=True)
-        result = subprocess.run(
+
+        install_prefix = (
             [
+                uv_helper.find_uv_bin(),
+                "pip",
+                "install",
+            ]
+            if uv_helper.should_use_uv()
+            else [
                 "pip",
```

Comment:
```suggestion
            else [
                sys.executable,
                "-m"
                "pip",
```

I missed this in my previous review above, from #58